controllers/feeds/reactionList.js

```diff
@@ -12,15 +12,8 @@ module.exports = async (req, res) => {
     const post = await Getstream.feed.getPlainFeedById(params.id);
     const myAnonymousId = await getAnonymUser(req.userId);
     const reaction = await reactionList(params.id, query.kind, query.limit);
-    let sortByDate = reaction.results.sort(
-      (a, b) => moment(a.created_at).unix() - moment(b.created_at).unix()
-    );
-    sortByDate = sortByDate.map((commentRes) => {
-      const sortComment = commentRes?.latest_children?.comment?.sort(
-        (a, b) => moment(a.created_at).unix() - moment(b.created_at).unix()
-      );
-      return {...commentRes, latest_children: {comment: sortComment}};
-    });
+
+    let level1Comments = buildComment(1, reaction.results);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `buildComment` function is being used here but it's not clear what the `1` argument represents. It would be better to use a constant with a meaningful name instead of a magic number for better readability and maintainability.
```diff
-    let level1Comments = buildComment(1, reaction.results);
+    const INITIAL_LEVEL = 1;
+    let level1Comments = buildComment(INITIAL_LEVEL, reaction.results);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->